lib/ferver.rb

```diff
@@ -4,6 +4,7 @@
 require_relative 'ferver/directory_not_found_error'
 require_relative 'ferver/file_id_request'
 require_relative 'ferver/file_list'
+require_relative 'ferver/found_file'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.